lib/zendesk_apps_tools/deploy.rb

```diff
@@ -126,5 +125,16 @@ def check_job(job_id)
     rescue Faraday::Error::ClientError => e
       say_error_and_exit e.message
     end
+
+    private
+
+    def zat_details(response)
+      zat_details = {
+        'subdomain' => @subdomain,
+        'username' => @username
+      }
+
+      zat_details.merge('app_id' => response['app_id']) if response['app_id']
+    end
```

Comment:
Another way I can do this is.
```
def zat_details(response)
  zat_details = {}
  zat_details['subdomain'] = @subdomain
  zat_details['username'] = @username
  zat_details['app_id'] = response['app_id']) if response['app_id']

  zat_details
```

This one ☝️ might be better

Reply:
Yeah the version in the comment seems more readable to me

Reply:
Agreed